app/components/UI/FiatOnRampAggregator/components/TransactionDetails.tsx

```diff
@@ -124,16 +127,29 @@ const TransactionDetails: React.FC<Props> = ({ order }: Props) => {
 	const { colors } = useTheme();
 	const styles = createStyles(colors);
 	const date = toDateFormat(order.data.createdAt);
-
+	const amountOut = order.data.fiatAmount - order.data.totalFeesFiat;
```

Comment:
Cast the arithmetic operation using the Number() method

Reply:
A general feedback here @VikChawla: try to use the shallow properties when possible, that's why we transform it. so we don't deal with nested data.

```ts
const aggregatorOrderToFiatOrder = (aggregatorOrder) => ({
	id: aggregatorOrder.id,
	provider: FIAT_ORDER_PROVIDERS.AGGREGATOR,
	createdAt: aggregatorOrder.createdAt,
	amount: aggregatorOrder.fiatAmount,
	fee: aggregatorOrder.totalFeesFiat,
	cryptoAmount: '',
	cryptoFee: '',
	currency: aggregatorOrder.fiatCurrency?.symbol || '',
	cryptocurrency: aggregatorOrder.cryptoCurrency?.symbol || '',
	network: aggregatorOrder.cryptoCurrency?.network?.chainId || '',
	state: aggregatorOrderStateToFiatOrderState(aggregatorOrder.status),
	account: aggregatorOrder.walletAddress,
	txHash: aggregatorOrder.txHash,
	data: aggregatorOrder,
});```